lib/Conversion/TorchOnnxToTorch/Patterns.cpp

```diff
@@ -33,12 +33,8 @@ LogicalResult OnnxCustomOpConversionPattern::matchAndRewrite(
   // overrides the function's domainVersion and will be used for matching later
   // here.
   if (auto attr = op->getAttrOfType<IntegerAttr>("torch.onnx_meta.version")) {
-    if (auto type = dyn_cast<IntegerType>(attr.getType())) {
-      if (type.isSigned()) {
-        opDomainVersion =
-            op->getAttrOfType<IntegerAttr>("torch.onnx_meta.version").getSInt();
-      }
-    }
+    assert(cast<IntegerType>(attr.getType()).isSigned());
```

Comment:
I agree with escaping from here if the version isn't signed, but I guess we should just refuse to lower the operator (`notifyMatchFailure`) instead of exiting entirely. What do you think?

Reply:
Hm, not really sure. I think the input op is actually broken (so probably there should be a verifier on the op?), so it is basically not possible to write a different lowering that handles it (it is not a shortcoming of the lowering really).

Reply:
Right. A verifier looks like a good solution.